### ISSUE TITLE
Fedora 34 virtual machines don't load grub config

### ISSUE THREAD

**Issue by ganto** [lxc/lxc-ci#279]
When trying to boot a Fedora 34 virtual machine (cloud and non-cloud version) the boot will stop a grub command prompt as it doesn't seem to load the `grub.cfg`:

```
$ lxc launch images:fedora/34/cloud --profile virtualmachine --vm --console
Creating the instance
[...]
To detach from the console, press: <ctrl>+a q

grub> 
```
It's then possible to start the VM by manually pointing grub to the correct configuration file:
```
grub> configfile (hd0,gpt2)/boot/grub2/grub.cfg
```
It seems that grub is not installed properly.

**Comment by ganto** [comment]
Is probably related to [Fedora 34: Unify the GRUB configuration files location across all supported architectures](https://fedoraproject.org/wiki/Changes/UnifyGrubConfig)

**Comment by stgraber** [comment]
Definitely sounds related

**Comment by monstermunchkin** [comment]
This was fixed in #288
